code/network/multimsgs.cpp

```diff
@@ -4047,8 +4082,9 @@ void send_netplayer_slot_packet()
 			ADD_DATA(stop);
 			ADD_SHORT(Net_players[idx].player_id);  
 			ADD_USHORT(Objects[Net_players[idx].m_player->objnum].net_signature);
-			ADD_INT(Net_players[idx].p_info.ship_class);
-			ADD_INT(Net_players[idx].p_info.ship_index);			
+			ADD_SHORT(static_cast<short>(Net_players[idx].p_info.ship_class));
+			val = static_cast<char>(Net_players[idx].p_info.ship_index);
+			ADD_DATA(val);
```

Comment:
Seems odd that index is smaller than ship class

Reply:
The index is for loadout and is < 12. The class is the Ship_info index. It confused me too.